docs/content/technical-specifications.md

```diff
@@ -23,7 +23,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
-| 3.0.1 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
+| 3.0.2 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
```

Comment:
Will re-add 3.0.1 versions

Reply:
You don't need to! We just keep the latest of each major version in this table

Reply:
I think we will have to update the helm chart and operator version though.

Reply:
think we can do another once IC is out